corehq/apps/accounting/models.py

```diff
@@ -964,6 +1026,10 @@ def renew_subscription(self, date_end=None, note=None, web_user=None,
             date_start=self.date_end,
             date_end=date_end,
         )
+        if type is not None:
```

Comment:
agree it's not the best name.  I used ```subscription_type``` in the code where I wanted to avoid replacing the ```type``` function, but didn't like introducing a ```Subscription``` parameter with the word "subscription" in it.  Since it's only accessed outside of here as ```subscription.type``` the namespacing shouldn't be a problem elsewhere where it's used